site/docs/.vuepress/styles/index.scss

```diff
@@ -6,9 +6,52 @@
 }
 
 html.dark {
+  --c-bg-lighter: #2b313a;
   --c-brand-light: #138cb3;
-  --c-tip: #1583a7;
   --c-brand: #009dca;
+  --c-tip: #1583a7;
+  --c-tip-title: var(--c-text-accent);
```

Comment:
This makes tip title appear in bright blue color (as opposed to text color by default)

Tips are not consistent with danger and warning blocks, but we can fix it. Anyway, that should be moved to a separate PR, as it requires changes to light theme too

Reply:
I believe it will be more appealing if the title text and accent color are the same color. I even considered modifying the container's background color to match the accent color, as with the warning and danger containers. What is your opinion?